drivers/usb/gadget/xaptum/f_psock_gadget.c

```diff
@@ -193,10 +232,18 @@ static struct usb_ss_ep_comp_descriptor ss_psock_sink_comp_desc = {
 
 static struct usb_descriptor_header *ss_psock_descs[] = {
         (struct usb_descriptor_header *) &psock_intf,
-        (struct usb_descriptor_header *) &ss_psock_source_desc,
-        (struct usb_descriptor_header *) &ss_psock_source_comp_desc,
-        (struct usb_descriptor_header *) &ss_psock_sink_desc,
-        (struct usb_descriptor_header *) &ss_psock_sink_comp_desc,
+
+        (struct usb_descriptor_header *) &f_psock_ss_bulk_source_desc,
+        (struct usb_descriptor_header *) &f_psock_ss_bulk_source_comp_desc,
+
+        (struct usb_descriptor_header *) &f_psock_ss_bulk_sink_desc,
+        (struct usb_descriptor_header *) &f_psock_ss_bulk_sink_comp_desc,
+
+        (struct usb_descriptor_header *) &f_psock_ss_ctrl_sink_desc,
+        (struct usb_descriptor_header *) &f_psock_ss_ctrl_comp_desc,
+
+        (struct usb_descriptor_header *) &f_psock_ss_ctrl_source_desc,
+        (struct usb_descriptor_header *) &f_psock_ss_ctrl_comp_desc,
```

Comment:
In the `strings_psock[]` table below,
- change `strings_psock[]` to `scm_string_defs[]`.
- change "psock interface" to "USB Socket Control Model (SCM)"

Chnage `stringtab_psock` to `scm_string_table`.

Change `psock_strings[]` to `scm_strings[]`.